src/udisksconfigmanager.c

```diff
@@ -187,6 +187,8 @@ udisks_config_manager_constructed (GObject *object)
         {
           udisks_debug ("No 'modules' found in configuration file");
           manager->modules = NULL;
+          g_error_free (error);
+          error = NULL;
```

Comment:
I think [g_clear_error](https://developer.gnome.org/glib/stable/glib-Error-Reporting.html#g-clear-error) is more appropriate here.